spec/models/preservation_copy_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe PreservationCopy, type: :model do
+  let!(:preservation_object2) do
+    PreservedObject.create!(druid: 'ab123cd45679', version: 1, preservation_policy: 'keepit', size: 1)
```

Comment:
Do we expect that 'preservation_policy' will be a small finite set, and if so does it make sense to represent it as such?

Reply:
similar to endpoint type, there'll be a policy definition table that this will point to (see #46).  so i'm inclined to leave this as-is and revisit it when we work on #46 (which should happen either later this week, or first thing next week).

Reply:
👍